includes/class-freemius.php

```diff
@@ -8514,7 +8515,7 @@ function _deactivate_plugin_hook() {
                     unset( $this->_storage->sticky_optin_added_ms );
                 }
 
-                if ( ! empty( $storage_keys_for_removal ) ) {
+                if ( count( $storage_keys_for_removal ) > 0 ) {

```

Comment:
@fajardoleo This is a fail safe for the `foreach ( $storage_keys_for_removal as $key ) {`

Reply:
Fail-safe for the `foreach` 😮 ? `foreach` works just fine with an empty array. Anyhow, I am cool with this check, since there was an `empty` check anyway.